<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pengaturan</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icon/</normaloff>:/icon/</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Warna Elektroda</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>FP1</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>F3</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>F4</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_1">
         <property name="text">
          <string>FP2</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>C3</string>
         </property>
        </widget>
        </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>C4</string>
         </property>
        </widget>
        </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>P3</string>
         </property>
        </widget>
        </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>P4</string>
         </property>
        </widget>
        </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>O1</string>
         </property>
        </widget>
        </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>O2</string>
         </property>
        </widget>
        </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>F7</string>
         </property>
        </widget>
        </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>F8</string>
         </property>
        </widget>
        </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>T3</string>
         </property>
        </widget>
        </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>T4</string>
         </property>
        </widget>
        </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>T5</string>
         </property>
        </widget>
        </item>
       <item row="15" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>T6</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="fp2_colour"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="f3_colour"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="f4_colour"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="fp1_colour"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="c3_colour"/>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="c4_colour"/>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="p3_colour"/>
        </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="p4_colour"/>
        </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="o1_colour"/>
        </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="o2_colour"/>
        </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="f7_colour"/>
        </item>
       <item row="11" column="1">
        <widget class="QLineEdit" name="f8_colour"/>
        </item>
       <item row="12" column="1">
        <widget class="QLineEdit" name="t3_colour"/>
        </item>
       <item row="13" column="1">
        <widget class="QLineEdit" name="t4_colour"/>
        </item>
       <item row="14" column="1">
        <widget class="QLineEdit" name="t5_colour"/>
        </item>
       <item row="15" column="1">
        <widget class="QLineEdit" name="t6_colour"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Warna Pita Frekuensi</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Gamma</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Alpha</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="beta_colour"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Delta</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="gamma_colour"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="alpha_colour"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Beta</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Theta</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="theta_colour"/>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="delta_colour"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>